WordPress/src/main/AndroidManifest.xml

```diff
@@ -113,7 +113,18 @@
         <activity
             android:name=".ui.accounts.LoginActivity"
             android:theme="@style/LoginTheme.TransparentSystemBars"
-            android:windowSoftInputMode="adjustResize" />
+            android:windowSoftInputMode="adjustResize"
+            android:exported="true">
```

Comment:
I think you're missing `android:launchMode="singleTask"`. Without that, a new Activity is created after the OAuth flow is approved, and you can navigate back to it (that shouldn't be possible).


https://github.com/user-attachments/assets/4869b516-3810-42eb-acdb-c36ea517b52e

`singleTop` does work as well in most browsers except Firefox. So `singleTask` is the safe option here.


Once you add that you should handle the `onNewIntent` properly.